.eslintrc.js

```diff
@@ -7,15 +7,21 @@ module.exports = {
     "import/extensions": [".mjs", ".js"],
     "import/resolver": {
       node: {
-        extensions: [".mjs", ".ts"],
+        extensions: [".mjs", ".ts", ".js", ".jsx"],
+        moduleDirectory: [
+          "src",
+          "node_modules",
+          "e2e/**/node_modules",
+          "examples/**/node_modules",
```

Comment:
These last two should be in overrides, for each specific subtree, so one for e2e and one for examples.